src/cactus/progressive/cactus_progressive.py

```diff
@@ -423,10 +437,6 @@ def main():
     parser.add_argument("outputHal", type=str, help = "Output HAL file")
 
     #Progressive Cactus Options
-    parser.add_argument("--database", dest="database",
-                      help="Database type: tokyo_cabinet or kyoto_tycoon"
-                      " [default: %(default)s]",
-                      default="kyoto_tycoon")
```

Comment:
please leave in the --database option for when we start migrating to redis or something else; although remvoing cabinet is good.  Maybe make it a choices argument?